app/views/layout.erb

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Chitter</title>
+</head>
+<body>
+<% if current_user %>
```

Comment:
You have some logic here for seeing if there is a user but in order to log in i had to manually type out the url. Maybe you should have given me a link to click on so I can easily sign in/up when required.